server/app/lib/actions/fusor/deployment/cloud_forms/wait_for_console.rb

```diff
@@ -32,12 +32,16 @@ def done?
 
           def invoke_external_task
             ::Fusor.log.debug "================ CloudForms::WaitForConsole invoke_external_task method ===================="
-            is_up? ::Fusor::Deployment.find(input[:deployment_id]).cfme_address
+            deployment = ::Fusor::Deployment.find(input[:deployment_id])
+            is_up? deployment.cfme_rhv_address if deployment.cfme_rhv_address
+            is_up? deployment.cfme_osp_address if deployment.cfme_osp_address
```

Comment:
This used to return on the is_up? but now it returns fromt he second call.  if you've installed both, and rhv is down but osp is up, this will return true.  Should this return rhv && osp?